model/model.go

```diff
@@ -83,6 +83,7 @@ type ColumnInfo struct {
 	types.FieldType     `json:"type"`
 	State               SchemaState `json:"state"`
 	Comment             string      `json:"comment"`
+	ExplictedCollation  bool        `json:"explicted_collation"`
```

Comment:
Any other **better variable name** ? I think this name is not good enough, but I cannot find a better name(avoid too long).
And I modeled the variable name of MYSQL(In MYSQL, this name is `column_has_explicit_collation`).